bundles/org.openhab.binding.tapocontrol/src/main/java/org/openhab/binding/tapocontrol/internal/structures/TapoDeviceConfiguration.java

```diff
@@ -55,7 +55,7 @@ public void loadSettings() {
         try {
             Configuration config = this.device.getConfiguration();
             this.ipAddress = config.get(CONFIG_DEVICE_IP).toString();
-            this.pollingInterval = Integer.valueOf(config.get(CONFIG_UPDATE_INTERVAL).toString());
+            this.pollingInterval = (int) Double.parseDouble(config.get(CONFIG_UPDATE_INTERVAL).toString());
```

Comment:
This work-around should not be needed in 3.4 as fixed by openhab/openhab-core#3024. See also #13112.